src/components/Stepper/StepperSeparator.tsx

```diff
@@ -0,0 +1,19 @@
+import {ChevronRight} from '@gravity-ui/icons';
+
+import {Icon} from '../Icon';
+
+import {b} from './utils';
+
+import './Stepper.scss';
+
+type StepperSeparatorProps = {
+    separator?: React.ReactNode;
+};
+
+export const StepperSeparator = ({separator}: StepperSeparatorProps) => {
+    return (
+        <div className={b('separator')} aria-hidden={true}>
+            {separator ?? <Icon data={ChevronRight} />}
```

Comment:
We should use `useDirection` to support RTL and change the default icon to `ChevronLeft`